Diagnostic/diagnostic.py

```diff
@@ -214,8 +214,8 @@ def check_for_supported_waagent_and_distro_version():
     :return: True iff so.
     """
     for notsupport in ('WALinuxAgent-2.0.5', 'WALinuxAgent-2.0.4', 'WALinuxAgent-1'):
-        code, str_ret = waagent.RunGetOutput("grep 'GuestAgentVersion.*" + notsupport + "' /usr/sbin/waagent",
-                                             chk_err=False)
+        code, str_ret = RunGetOutput("grep 'GuestAgentVersion.*" + notsupport + "' /usr/sbin/waagent",
+                                             should_log=False)
```

Comment:
RunGetOutput() is referring to DistroSpecific -> log_run_get_output. Pls refer to: https://github.com/Azure/azure-linux-extensions/blob/master/Diagnostic/diagnostic.py#L88

the above line is executed before any call to RunGetOutput()

should_log argument has same meaning as chk_err